lib/lita/handlers/room_queue.rb

```diff
@@ -8,6 +8,10 @@ def initialize(name, redis)
         @redis = redis
       end
 
+      def to_s
+        return self.queue.join('\n')
```

Comment:
Drop the explicit return here and rebase the `pop/shift` change out and I'd be happy to merge! (Since I already fixed that on master, whoops!)